packages/containers.nix

```
@@ -198,5 +198,9 @@ containers
   push-node-installer-kata =
     pushOCIDir "push-node-installer-kata" pkgs.kata.contrast-node-installer-image
       "v${pkgs.contrast.version}";
+  push-node-installer-kata-gpu = pushOCIDir "push-node-installer-kata-gpu" (
+    pkgs.kata.contrast-node-installer-image.override
+    { withGPU = true; }
```

```suggestion
    pkgs.kata.contrast-node-installer-image.gpu
```
Can't we use the passedthru gpu attribute here?